README.md

```diff
@@ -95,6 +95,7 @@ The following devices are supported by this version of liquidctl.  See each guid
 | AIO liquid cooler | [Corsair Hydro H100i Pro XT, H115i Pro XT, H150i Pro XT](docs/corsair-platinum-pro-xt-guide.md) | USB HID | <sup>_e_</sup> |
 | AIO liquid cooler | [Corsair iCUE H100i, H115i, H150i Elite Capellix](docs/corsair-commander-core-guide.md) | USB HID | <sup>_ep_</sup> |
 | AIO liquid cooler | [EVGA CLC 120 (CL12), 240, 280, 360](docs/asetek-690lc-guide.md) | USB | <sup>_Z_</sup> |
+| AIO liquid cooler | [MSI MPG Series CORELIQUID K360](docs/msi-mpg-series-coreliquid-guide.md) | USB HID | <sup>_penx_</sup> |
```

Comment:
MSI, as most other vendors, abuses all caps and long product names.

When that happens, we intentionally deviate from the offical name and adopt something a bit more sensible; in this case, I like "MSI MPG Coreliquid K360".